bin.src/ci_imsim_run.py

```diff
@@ -29,7 +29,7 @@ class ImsimWriteCuratedCalibrations(WriteCuratedCalibrations):
     instrumentName = INSTRUMENT_NAME
 
 
-ciRunner.register("skymap", 3)(RegisterSkyMap)
+ciRunner.register("skymap", index_command := index_command + 1)(RegisterSkyMap)
```

Comment:
I'm not entirely sure what this change is about and it doesn't seem to be related to the commit message. Can you put it on a different commit so I can understand why it's needed?

Reply:
It's not strictly necessary; the commands were running in the right order as-is. However, I had originally hard-coded the numerical index of each command, which made it annoying to maintain integer increments* when adding/removing commands; I just happened to miss changing the index for this one command.

*apparently they are floats and so one could add decimals instead or specify the order completely differently from the function definition order but that would be unnecessary and confusing.